tests/robustness/watch.go

```diff
@@ -110,9 +113,7 @@ func watchMember(ctx context.Context, t *testing.T, c *clientv3.Client, maxRevis
 				c.RequestProgress(ctx)
 			}
 			if resp.Err() == nil {
-				// using time.Since time-measuring operation to get monotonic clock reading
-				// see https://github.com/golang/go/blob/master/src/time/time.go#L17
```

Comment:
Can we keep the comment? :/

Reply:
Done and added in all places referencing baseTime.